src/index.js

```diff
@@ -1,11 +1,13 @@
 import createAction from './createAction';
 import handleAction from './handleAction';
 import handleActions from './handleActions';
+import combineActions from './combineActions';
 import createActions from './createActions';
 
 export {
   createAction,
   createActions,
   handleAction,
+  combineActions,
```

Comment:
Ugh, sorry, I should have been catching these. It's been a while since the PR, and I was on vacation 😓 

5dd2f29